monai/transforms/utils.py

```diff
@@ -157,3 +156,108 @@ def one_hot(labels, num_classes):
     onehot = y[labels.flatten()]
 
     return onehot.reshape(tuple(labels.shape) + (num_classes,)).astype(labels.dtype)
+
+
+def create_grid(spatial_size, homogeneous=True):
+    """
+    compute a `spatial_size` mesh.
+    """
+    ranges = [np.linspace(-(d - 1.) / 2., (d - 1.) / 2., int(d)) for d in spatial_size]
+    coords = np.asarray(np.meshgrid(*ranges, indexing='ij'), dtype=float)
+    if not homogeneous:
+        return coords
+    return np.concatenate([coords, np.ones_like(coords[0:1, ...])])
+
+
+def create_rotate(spatial_dims, radians):
+    """
+    create a 2D or 3D rotation matrix
+    Args:
+        spatial_dims (int): spatial rank
+        radians (float or a sequence of floats): rotation radians
+    """
+    if spatial_dims == 2:
+        if isinstance(radians, (list, tuple)):
+            radians = radians[0]
+        return np.array([[np.cos(radians), -np.sin(radians), 0.], [np.sin(radians), np.cos(radians), 0.], [0., 0., 1.]])
+
+    if spatial_dims == 3:
+        affine = None
+        if len(radians) >= 1:
+            affine = np.array([
+                [1., 0., 0., 0.],
+                [0., np.cos(radians[0]), -np.sin(radians[0]), 0.],
```

Comment:
The cos and sin calculations each appear twice, I don't think we can expect these to be optimized together so they should be pre-calculated before this line and referenced in the array.